### PR TITLE
Add SQL support section to Kudu connector

### PR DESCRIPTION
**Pull request by Jessie212**

### CODE REVIEW

File: docs/src/main/sphinx/connector/kudu.rst [hunk 1]
```diff
@@ -228,67 +227,53 @@ The data types of Trino and Kudu are mapped as far as possible:
...
+``UPSERT``.
+
+ .. include:: sql-delete-limitation.fragment
+
+.. _kudu-create-schema:
+
+Creating schemas
+^^^^^^^^^^^^^^^^
+
+``CREATE SCHEMA`` is only allowed if schema emulation is enabled.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/kudu.rst:264]: For both this and the DROP SCHEMA section below, these should link to the "Behavior with schema emulation" section above. Suggest adding an anchor to that above section and linking with a :ref:

Comment (hashhar) [reply docs/src/main/sphinx/connector/kudu.rst:264]: Good point, Kudu doesn't have real schemas and people need to understand the schema emulation thing before they go around playing with schemas.

File: docs/src/main/sphinx/connector/kudu.rst [hunk 2]
```diff
@@ -228,67 +229,55 @@ The data types of Trino and Kudu are mapped as far as possible:
...
+
+.. include:: sql-delete-limitation.fragment
+
+.. _kudu-create-schema:
+
+Creating schemas
+^^^^^^^^^^^^^^^^
+
+``CREATE SCHEMA`` is only allowed if schema emulation is enabled. See the
+:ref:`Behavior-with-schema-emulation` section.
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/kudu.rst:267]: Does this build without errors? The case doesn't match - would make these refs lowercase to match

File: docs/src/main/sphinx/connector/kudu.rst [hunk 3]
```diff
@@ -228,67 +229,55 @@ The data types of Trino and Kudu are mapped as far as possible:
...
+
+.. _kudu-insert:
+
+Inserting into tables
+^^^^^^^^^^^^^^^^^^^^^
+
+``INSERT INTO ... values`` and ``INSERT INTO ... select`` behave like
+``UPSERT``.
+
+.. include:: sql-delete-limitation.fragment
```
Comment (jhlodin) [inline docs/src/main/sphinx/connector/kudu.rst:259]: Quick nit, can you move this include above the kudu-insert section and anchor? Having this here looks, at a glance, like the fragment is part of the "inserting into tables" section.

**Review by jhlodin — APPROVED** [review]
LGTM with comments

File: docs/src/main/sphinx/connector/kudu.rst (hunk 2 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/kudu.rst]: It does build without errors. I matched the case to the other ref links.

File: docs/src/main/sphinx/connector/kudu.rst (hunk 3 above)
Comment (Jessie212) [reply docs/src/main/sphinx/connector/kudu.rst:259]: @mosabua and I moved it down to match the order of the list.

Comment (jhlodin) [reply docs/src/main/sphinx/connector/kudu.rst:259]: Never mind then!

**Review by rosewms — APPROVED** [review]
LGTM

**Review by hashhar — APPROVED** [review]

**Pull request merged**